module/Activity/src/Activity/Controller/AdminApprovalController.php

```diff
@@ -47,7 +22,17 @@ public function viewAction()
         $id = (int) $this->params('id');
         $queryService = $this->getServiceLocator()->get('activity_service_activityQuery');
 
-        /** @var $activity Activity*/
+        $acl = $translator = $this->getServiceLocator()->get('activity_acl');
```

Comment:
$acl = $translator = $this.... ?
This seems like a bug as well.
Try assigning either to $acl or $translator